src/langs/en.json

```
@@ -106,5 +106,10 @@
   "MEMBER_STORAGE_PARENT_FOLDER": "Parent folder",
   "NO_PARENT": "-",
   "STORAGE_FILES_ERROR": "Oops, an error occurred while fetching storage files. Please try again later.",
-  "STORAGE_FILES_EMPTY": "You didn't upload any file."
+  "STORAGE_FILES_EMPTY": "You didn't upload any file.",
+  "EXPORT_INFORMATIONS_TITLE": "Export data",
+  "EXPORT_INFORMATIONS_DESCRIPTION": "You can download a copy of your data",
+  "EXPORT_INFORMATIONS_BUTTON_TEXT": "Start exporting",
+  "EXPORT_SUCCESS_MESSAGE": "Your request has been sent. An email will be sent to you in several minutes.",
```

I'd write something like the following (it is only a suggestion :) ).

```suggestion
  "EXPORT_SUCCESS_MESSAGE": "You have successfully requested your data. You will receive an e-mail in a few minutes.",
```